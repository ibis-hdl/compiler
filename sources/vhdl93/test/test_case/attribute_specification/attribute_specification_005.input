attribute IMPLEMENTATION of G1: group is "74LS152";

