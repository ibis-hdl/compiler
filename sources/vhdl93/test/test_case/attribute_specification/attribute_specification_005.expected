(attribute_specification
  (simple_name
    IMPLEMENTATION
  )
  (entity_specification
    (entity_name_list<v>
      (entity_designator
        (entity_tag<v>
          (simple_name
            G1
          )
        )
      )
    )
    (keyword
      GROUP
    )
  )
  (expression
    (relation
      (shift_expression
        (simple_expression
          term: (term
            (primary<v>
              (literal<v>
                (string_literal
                  74LS152
                )
              )
            )
          )
        )
      )
    )
  )
)
